python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -138,9 +138,8 @@ def checkHostName():
         except:
             pass
 
-        self.syscfg.svo.stopService("cloudstack-management")
-
         if self.syscfg.env.noStart == False:
+            self.syscfg.svo.stopService("cloudstack-management")
```

Comment:
This is the original behavior of the script, before `--no-start` was added. When `--no-start` is NOT specified, the script tries to make sure that the service is stopped before starting it again.